examples/using-remark/src/pages/excerpt-example.js

```diff
@@ -0,0 +1,95 @@
+import React from "react"
+import Link from "gatsby-link"
+import styles from "../styles"
+import presets from "../utils/presets"
+import { rhythm, scale } from "../utils/typography"
+
+class Index extends React.Component {
+  render() {
+    const posts = this.props.data.allMarkdownRemark.edges
+    console.log(posts)
+
+    return (
+      <div>
+        <div>
+          <h1
+            css={{
+              ...scale(4 / 5),
+              fontWeight: `800`,
+              marginBottom: rhythm(2),
+            }}
+          >
+            This example demonstrates{` `}
```

Comment:
Change the introduction to make it clear that people are seeing two excerpt styles? Perhaps duplicate some of the content from the excerpt explanation page.

Reply:
Updated the header and some of the excerpts to make it a bit more clear. How does this look?